Runtime/Models/IObjectsTexturesRecord.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ReupVirtualTwin.models
+{
+    public interface IObjectsTexturesRecord
```

Comment:
A pedantic suggestion 
```suggestion
    public interface IObjectTexturesRecord
```